README.md

```diff
@@ -706,6 +706,7 @@ Usage: `nerdctl stop [OPTIONS] CONTAINER [CONTAINER...]`
 
 Flags:
 - :whale: `-t, --time=SECONDS`: Seconds to wait for stop before killing it (default "10")
+  - Tips: If the init process in container is exited after receiving SIGTERM or exited before the time you specified, the container will be exited simultaneously
```

Comment:
I don't get what "simultaneously" means. Maybe s/simultaneously/immediately/ ?

Reply:
Yes! I will update the word